src/app/shared/components/navbar/navbar.component.html

```diff
@@ -1,19 +1,19 @@
 <div class="respmenu">
-  <h1 routerLink="/home" routerLinkActive="router-link-active" >
+  <h1 routerLink="/" routerLinkActive="router-link-active" >
     <i  class="fa-solid fa-building-columns"></i>
     &emsp; Banco Herrera
   </h1>
-  <input type="checkbox" >
+  <input type="checkbox" #ckeckbox >
   <i class="fas fa-bars"></i>
   <i class="fas fa-times"></i>
 
   <nav>
 
     <ul>
-      <a routerLink="home" >Home</a>
+      <a routerLink="/" (click)="ckeckbox.checked = false">Home</a>
```

Comment:
aquí lo que pasa si miras el css, dice que sea dislay none si el checkbox no es checked, y bueno cuando le das click al checkbox pues se pone en checked, y si está en checked se pone en not checked,

sabiendo esto, lo que hacemos entonces es crear una referencia a este checkbo que se llama "checkbox" xD y entonces le decimos que cuando se da click en home, o en about, entonces será not checked dicho de otra forma: checkbox.checked = false